### PR TITLE
Extend subctl validate all to include even firewall commands

### PR DESCRIPTION
**Pull request by sridhargaddam**
Signed-Off-by: Sridhar Gaddam <sgaddam@redhat.com>

### CODE REVIEW

**Comment by sridhargaddam** [comment]
@Jaanki addressed your review comments in this PR :)

File: pkg/subctl/cmd/validate_all.go [hunk 1]
```diff
@@ -61,5 +61,13 @@ func validateAll(cmd *cobra.Command, args []string) {
 		checkOverlappingCIDRs(item, submariner)
 		fmt.Println()
 		validateKubeProxyModeInCluster(item.config, item.clusterName)
+		fmt.Println()
+		validateFirewallMetricsConfigWithinCluster(item.config, item.clusterName)
+		fmt.Println()
+		validateVxLANConfigWithinCluster(item.config, item.clusterName, submariner)
+		fmt.Println()
+		fmt.Printf("Validating the tunnels is skipped as it requires two kubeconfigs." +
+			" Please run \"subctl validate firewall tunnel\" command manually.\n")
```
Comment (tpantelis) [inline pkg/subctl/cmd/validate_all.go]:
```suggestion
		fmt.Printf("Skipping  tunnel firewall validation as it requires two kubeconfigs." +
			" Please run \"subctl validate firewall tunnel\" command manually.\n")
```

Comment (sridhargaddam) [reply pkg/subctl/cmd/validate_all.go]: Done

**Review by maayanf24 — APPROVED** [review]

**Review by mangelajo — APPROVED** [review]

**Review by Jaanki — APPROVED** [review]
Thank you @sgaddam for addressing the comments. This looks good.

**Pull request merged**